logger.go

```diff
@@ -0,0 +1,17 @@
+package kafka
+
+// Logger interface API for log.Logger
+type Logger interface {
+	Printf(string, ...interface{})
+}
+
+// LoggerFunc is a bridge between Logger and any third party logger
+// Usage:
+// l := NewLogger() // some logger
```

Comment:
If you want this test to be formatted as code in the godoc it should be indented one level.

Here's a cheatsheet I used to learn these tricks: https://godoc.org/github.com/fluhus/godoc-tricks

Reply:
Cool, thanks, fixed